src/components/Transactions.vue

```diff
@@ -1,16 +1,33 @@
 <template>
   <div class="transactions">
-    <TransactionList :transactions="transactions" />
+    <TransactionList class="list" :transactions="transactions" />
+    <Balances :available="available" :timelocked="timelocked" :unconfirmed="unconfirmed" :total="total" />
   </div>
 </template>
 
 <script>
+import Balances from './Balances'
 import TransactionList from './TransactionList'
 
 export default {
   name: 'Transactions',
   components: {
     TransactionList,
+    Balances,
+  },
+  computed: {
+    available () {
+      return this.$store.state.wallet.wallet.balances.available
```

Comment:
What.

Reply:
This will be less verbose when we close #717